test/test_adjoint.jl

```diff
@@ -10,6 +10,7 @@ parsed_args = parse_commandline()
 nlayer = parsed_args["nlayer"]
 tti = parsed_args["tti"]
 fs =  parsed_args["fs"]
+isic =  parsed_args["isic"]
```

Comment:
isic isn't one of the  command line arguments in any of the test setup so none of `isic=true` this is tested on;ly the default case is tested.